content/blog/posts/o3de-user-defined_properties-2022-03.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "O3DE User Defined Properties"
+date: 2022-03-08
+slug: blog-udp
+author: Allen Jackson
+blog_img: "/images/blog/udp-intro/editor_def_mat_port.png"
+full_img: "/images/blog/udp-intro/editor_def_mat_port.png"
+---
+
+# O3DE User Defined Properties
+
+Artists store metadata in source scenes (e.g. blend, ma, 3ds file extensions) to user defined properties (UDP) on scene nodes. This user defined property metadata can be exported into source scene asset files (e.g. FBX or glTF). Game teams use this UDP metadata to tag mesh data for engine purposes such as splitting up meshes into level of details (LODs), flagging a simplified mesh as a collision mesh to use as a trigger or hit detection, and assigning scene features like render materials to engine models.
```

Comment:
This appears to be missing something "Artists store metadata in source scene files to [define? setup?] user defined properties (UDP) on scene nodes".